### ISSUE TITLE
[DESIGN] Group Facility Locator COVID content and links under a heading

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#27447]
## User story
As a Veteran who uses assistive technology, I am confused to hear COVID-related links organized under a heading about finding a facility. 

## Issue Description
Assistive technology users already spend extra time understanding and orienting themselves on a page. We can improve this experience by grouping links and content with logically ordered heading levels to create an understandable hierarchy of the page. 

**Reference:** [Facility Locator short-term recommendations #3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/facility-locator/research/user-research/screenreader-usability-study/research-findings.md#on-facility-locator---short-term)

---
## Tasks
- [ ] Move the COVID content and links into an alert region that can be announced as such.

## Acceptance Criteria
- [ ] Assistive technology users are able to scan content headings in order to find and understand the content in context  and navigate the page more quickly.

**Comment by davidconlon** [comment]
@mmiddaugh 
This recommendation is outdated as we have updated the text/links. 
We now only have one link that pertains to masking at our locations.
I don't feel the need to move this into an alert/run this through JTF.

**Comment by mmiddaugh** [comment]
COVID-19 content cannot be skipped (Joint Task Force).
